posthog/hogql/database/database.py

```diff
@@ -211,7 +210,7 @@ def create_hogql_database(
             if "timestamp" not in tables[warehouse_modifier.table_name].fields.keys():
                 tables[warehouse_modifier.table_name].fields["timestamp"] = ExpressionField(
                     name="timestamp",
-                    expr=ast.Call(name="toDateTime", args=[ast.Field(chain=[warehouse_modifier.timestamp_field])]),
+                    expr=ast.Field(chain=[warehouse_modifier.timestamp_field]),
```

Comment:
We can check the type of the timestamp field and conditionally wrap it in a `toDateTime` if its not already a `DateTime` database field